modules/julia/gen/cpp_files/jlcv.hpp

```diff
@@ -2,9 +2,7 @@
 // It is subject to the license terms in the LICENSE file found in the top-level directory
 // of this distribution and at http://opencv.org/license.html
 // Copyright (C) 2020 by Archit Rungta
-
-
-// This header file contains some template deductions for type-mapping andconversion
+// This header files hacks into the mapping code of CxxWrap to support automatic conversion between types from OpenCV ad Julia
```

Comment:
Please keep at least one empty line to separate description from license+copyright header.

Looks like there is a typo in the end.

Reply:
Nice catch! Fixed it.